sklearnex/covariance/incremental_covariance.py

```diff
@@ -113,9 +113,9 @@ def _onedal_supported(self, method_name, *data):
         )
         return patching_status
 
-    def _onedal_finalize_fit(self):
+    def _onedal_finalize_fit(self, queue=None):
```

Comment:
How is queue obtained when called from line 136 or 147?

Reply:
it takes queue stored from partial_fit call in attributes of onedal estimator